phpstan-baseline.neon

```diff
@@ -15,6 +15,26 @@ parameters:
 			count: 1
 			path: src/Controller/BaseCrudController.php
 
+		-
+			message: "#^Cannot call method getName\\(\\) on array\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\.$#"
+			count: 1
+			path: src/Test/Controller/AdminControllerWebTestCase.php
+
+		-
+			message: "#^Parameter \\#1 \\$first of function Psl\\\\Type\\\\union expects Psl\\\\Type\\\\TypeInterface\\<array\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\>, Psl\\\\Type\\\\TypeInterface\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\> given\\.$#"
+			count: 1
+			path: src/Test/Controller/AdminControllerWebTestCase.php
+
+		-
+			message: "#^Parameter \\#1 \\$iterable of function Psl\\\\Dict\\\\map expects iterable\\<\\(int\\|string\\), array\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>, array\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField given\\.$#"
+			count: 1
+			path: src/Test/Controller/AdminControllerWebTestCase.php
+
+		-
+			message: "#^Parameter \\#2 \\$second of function Psl\\\\Type\\\\union expects Psl\\\\Type\\\\TypeInterface\\<array\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\|Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\>, Psl\\\\Type\\\\TypeInterface\\<array\\<array\\<Symfony\\\\Component\\\\DomCrawler\\\\Field\\\\FormField\\>\\>\\> given\\.$#"
+			count: 1
+			path: src/Test/Controller/AdminControllerWebTestCase.php
+
```

Comment:
PSL doesn't use phpstan, i will have to look into this and see what is the issue exactly.

psalm passes as is.